### DISCUSSION TITLE
Possible to add row click event on FluentDataGrid?

### DISCUSSION THREAD

**Discussion by skillcrushbinguyen** [microsoft/fluentui-blazor#1827] [Q&A]
I'm finding a way to add row click event on FluentDataGrid. But I can't find anything document talking about this. Any possible way to do this with FluentDataGrid ?

**Comment by dvoituron** [comment]
You can use the `OnRowFocus` attribute.

```razor
<FluentDataGrid OnRowFocus="@HandleRowFocus" ... > ... </FluentDataGrid>

@code {
    private void HandleRowFocus(FluentDataGridRow<YourItemType> row)
    {
        // ...
    }
}
```

**Reply by skillcrushbinguyen** [reply]
I need row click event because I want to open the menu on this row.

**Reply by vnbaaij** [reply]
Hi,

It seems you are new here. Welcome!

Instead of saying 'I need' (demanding, aggressive) use 'I would like' or 'My app's requirements are that I can' please. Thank you.

**Reply by skillcrushbinguyen** [reply]
Hi @vnbaaij , 
I’m so sorry this impolite. Because my native language doesn’t English. I will pay attention in the next time, again I so sorry. Thank you for give me advise
